shared/constants/config.js

```diff
@@ -80,6 +80,8 @@ export const urlToUsername = (url: {
 export const makeState: I.RecordFactory<Types._State> = I.Record({
   appFocused: true,
   appFocusedCount: 0,
+  appOutOfDateMessage: 'aaa',
+  appOutOfDateStatus: 'critical',
```

Comment:
Any reason not to use https://github.com/keybase/client/blob/59a25dce1e8758a890e50b2767fbd6625f838915/shared/constants/types/config.js#L42 ? It'd be nice to unify this with the one in widget.

Reply:
we should talk to max